### PR TITLE
feat: add support for olive release

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
### Description
This PR adds the plugin support for the olive release as a BTR migration effort.

### How to test
1. Install tutor olive version
2. Install this plugin branch
3. Turn on your tutor environment
4. Follow [the instructions](https://github.com/eduNEXT/tutor-contrib-codejail) to install and test the plugin

### CODE REVIEW

**Comment by MoisesGSalas** [comment]
Should we release a new version of [codejailservice](https://github.com/eduNEXT/codejailservice) before merging this?

**Comment by mariajgrimaldi** [comment]
> Should we release a new version of [codejailservice](https://github.com/eduNEXT/codejailservice) before merging this?

I created a 15.0.0 branch for the time being

File: tutorcodejail/templates/codejail/build/codejail/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-FROM docker.io/ubuntu:20.04 as minimal
+FROM docker.io/ubuntu:22.04 as minimal
```
Comment (MoisesGSalas) [inline tutorcodejail/templates/codejail/build/codejail/Dockerfile:1]: I don't think anything is written in stone, but maybe we should try to keep the same base Ubuntu across images?

Seems like both [discovery](https://github.com/overhangio/tutor-discovery/blob/olive/tutordiscovery/templates/discovery/build/discovery/Dockerfile#L1) and [ecommerce](https://github.com/overhangio/tutor-ecommerce/blob/olive/tutorecommerce/templates/ecommerce/build/ecommerce/Dockerfile#L1) are going to keep using 20.04.

Comment (mariajgrimaldi) [reply tutorcodejail/templates/codejail/build/codejail/Dockerfile:1]: This was Regis' recommendation

Comment (MoisesGSalas) [reply tutorcodejail/templates/codejail/build/codejail/Dockerfile:1]: He kind of backtracked on that: https://github.com/overhangio/tutor-notes/pull/18#discussion_r1019239148.

Comment (mariajgrimaldi) [reply tutorcodejail/templates/codejail/build/codejail/Dockerfile:1]: But this is already working. Let's not waste the effort 😭

**Review by MoisesGSalas — APPROVED** [review]

**Pull request merged**